apis/goharbor.io/v1alpha2/harborcluster_types.go

```diff
@@ -127,14 +134,16 @@ type MinIOSpec struct {
 	// Number of persistent volumes that will be attached per server
 	// +kubebuilder:validation:Minimum:=1
 	VolumesPerServer int32 `json:"volumesPerServer"`
-	// Version defines the MinIO Client (mc) Docker image version.
-	Version string `json:"version,omitempty"`
 	// VolumeClaimTemplate allows a user to specify how volumes inside a MinIOInstance
 	// +kubebuilder:validation:Optional
 	VolumeClaimTemplate corev1.PersistentVolumeClaim `json:"volumeClaimTemplate,omitempty"`
 	// If provided, use these requests and limit for cpu/memory resource allocation
 	// +kubebuilder:validation:Optional
 	Resources corev1.ResourceRequirements `json:"resources,omitempty"`
+
+	// +kubebuilder:validation:Optional
+	// Image name for the minio.
+	Image string `json:"image,omitempty"`
```

Comment:
It seems not just need `image` here, we also need to consider the imagePullPolicy and imagePullingSecrets.